com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeVolumeSceneData.cs

```diff
@@ -230,6 +266,45 @@ internal BakingSet CreateNewBakingSet(string name)
             return set;
         }
 
+        internal void CreateNewBakingState(string name)
+        {
+            bakingStates.Add(name);
+        }
+
+        internal void RemoveBakingState(int index)
```

Comment:
So this reassigns indices to the right states which is fine, but what about lingering references to old states (just ints afterall)?